apps/files/src/components/FileactionsTab.vue

```diff
@@ -33,6 +50,11 @@ export default {
         name: appId
       })
     },
+    downloadFiles (file) {
+      // TODO fix missing bearer Token / broken $client
+      console.log('js-owncloud-client issue - Bearer Token missing', file)
+      this.downloadFile(file)
```

Comment:
where is this code change coming from - should not have been touched ...

Reply:
This was the addition to #579. Any idea why Bearer is undefined?

Reply:
different topic -> different PR - don't mix changes

Reply:
undefined bearer could be related to the initialization process - we had this already in the search somewhere ....